### ISSUE TITLE
Flaky test: TreeView file modification tree-view:add-file ...

### ISSUE THREAD

**Issue by jasonrudolph** [atom/atom#15363]
Failed on CircleCI: https://circleci.com/gh/atom/atom/4764

```
Executing tree-view tests
Package tests failed for tree-view:
.....................................................................................................................................................................................................................................................F.................................................................--....................................................

TreeView
  file modification
    tree-view:add-file
      when a file is selected
        when the path without a trailing '/' is changed and confirmed
          when no file exists at that location
            it adds a file, closes the dialog, selects the file in the tree-view, and emits an event
              Expected 'test-dirnew-test-file.txttest-file.txt' to be 'new-test-file.txt'.
                at jasmine.Spec.<anonymous> (/Users/distiller/atom/node_modules/tree-view/spec/tree-view-package-spec.coffee:1816:81)

Finished in 99.977 seconds
363 tests, 1892 assertions, 1 failure, 2 skipped
```

**Comment by 50Wliu** [comment]
@jasonrudolph I thought you fixed this in https://github.com/atom/tree-view/pull/1156?

**Comment by jasonrudolph** [comment]
> I thought you fixed this in atom/tree-view#1156?

@50Wliu: Good catch. I _knew_ this looked familiar. 😅

atom/tree-view#1156 was released in tree-view v0.217.6. https://github.com/atom/atom/pull/15145 upgraded atom/atom to tree-view v0.217.6. That change exists on the `1.20-releases` branch and the `master` branch, but not the `1.19-releases` branch. This CI failure occurred on the atom `1.19-releases` branch, which is using tree-view v0.217.2:

https://github.com/atom/atom/blob/85b7cff330bf7f56ad163efb47e75a9e8154a320/package.json#L129

With that in mind, I'm gonna close this issue. We may continue to see these failures on the `1.19-releases` branch, but they should be resolved in newer versions of Atom.
